dnsdist: Add `NetmaskGroup:addNMG()` to merge Netmask groups (#13503)
PowerDNS/pdns: pdns/dnsdistdist/docs/reference/netmaskgroup.rst

    :param string mask: Add this mask, prefix with `!` to exclude this mask from matching.
    :param table masks: Adds the keys of the table to the :class:`NetmaskGroup`. It should be a table whose keys are :class:`ComboAddress` objects and whose values are integers. The integer values of the table entries are ignored. The table is of the same type as the table returned by the `exceed*` functions.

  .. method:: NetmaskGroup:addNMG(otherNMG)

    .. versionadded:: 1.9.0

>     Add one or more masks from an existing to this NMG.

Reviewer: ```suggestion
    Add all masks from an existing NMG to this NMG.
```
